src/deploy/functions/backend.ts

```diff
@@ -622,6 +623,7 @@ async function loadExistingBackend(ctx: Context & PrivateContextFields): Promise
       ctx.existingBackend.topics.push({
         id,
         project: specFunction.project,
+        labels: { deployment: "firebase-schedule" },
```

Comment:
Maybe turn "firebase-schedule" into an exported constant somewhere, since we use it in a few places across a few files.